code/game/objects/items.dm

```diff
@@ -586,3 +586,9 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
 					"<span class='userdanger'>You block the attack!</span>")
 		user.Stun(2)
 		return TRUE
+
+/obj/item/proc/update_slot_icon()

```

Comment:
This proc won't really be called, but is more resource intensive than /tg/'s because it regenerates all of the owners icons. We don't have their slot defines though, and this won't ever really be called right now.